boskos/janitor/janitor.go

```diff
@@ -55,6 +57,14 @@ func main() {
 		logrus.Fatal("--resource-type must not be empty!")
 	}
 
+	go func(boskos boskosClient) {
+		for range time.Tick(updateFrequency) {
+			if err := boskos.SyncAll(); err != nil {
```

Comment:
minor note: this now going to issue sequential update calls instead of in parallel? how long do these typically take? is this N<=20 resources or potentially more?
looks like it at least continues to the next resource on error?

Reply:
N will be <= # of goroutines, which is by default 20.

SyncAll locks, so it will block on other require/release actions.